protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -549,12 +445,10 @@ contract Storage {
  * @param fFirst The lowest active Fertilizer Id (start of linked list that is stored by nextFid).
  * @param fLast The highest active Fertilizer Id (end of linked list that is stored by nextFid).
  * @param bpf The cumulative Beans Per Fertilizer (bfp) minted over all Season.
- * @param deprecated_vestingPeriodRoots deprecated - removed in BIP-39 in favor of germination.
  * @param recapitalized The number of USDC that has been recapitalized in the Barn Raise.
  * @param isFarm Stores whether the function is wrapped in the `farm` function (1 if not, 2 if it is).
  * @param ownerCandidate Stores a candidate address to transfer ownership to. The owner must claim the ownership transfer.
  * @param wellOracleSnapshots A mapping from Well Oracle address to the Well Oracle Snapshot.
- * @param deprecated_beanEthPrice DEPRECATED - The price of bean:eth, originally used to calculate the incentive reward. Deprecated in favor of calculating using twaReserves.
  * @param twaReserves A mapping from well to its twaReserves. Stores twaReserves during the sunrise function. Returns 1 otherwise for each asset. Currently supports 2 token wells.
  * @param migratedBdvs Stores the total migrated BDV since the implementation of the migrated BDV counter. See {LibLegacyTokenSilo.incrementMigratedBdv} for more info.
```

Comment:
could we switch `migratedBdvs` and `twaReserves`? They look to be in the incorrect order.

Reply:
fixed in 7ea86b8f732de1bd4bea2313c197882e15f888b6